docs/introduction-to-smart-contracts.rst

```diff
@@ -207,9 +206,7 @@ The functions that make up the contract, and that users and contracts can call a
 
 The ``mint`` function sends an amount of newly created coins to another address.
 The :ref:`require <assert-and-require>` function call defines conditions that reverts all changes if not met.
-In this example, ``require(msg.sender == minter);`` ensures that only the creator of the contract can call ``mint``,
-and ``require(amount < 1e60);`` ensures a maximum amount of tokens: The function can be called very often,
-but due to gas costs, there is still a practical limit, and ``uint`` maximum value exceeds 1e77.
+In this example, ``require(msg.sender == minter);`` ensures that only the creator of the contract can call ``mint``.
```

Comment:
```suggestion
In this example, ``require(msg.sender == minter);`` ensures that only the creator of the contract can call ``mint``. Note that because of the default :ref:`Checked arithmetic <_unchecked>`, the transaction would revert if the expression ``balances[receiver] += amount;`` overflows, i.e., when ``balances[receiver] + amount`` in arbitrary precision arithmetic is larger than the maximum value of ``uint`` (``2**256 - 1``).
```

Reply:
@waded Already committed this change. I would say this change can now be merged if you can squash all the commits. Could you please squash. Also thanks for the PR.

Reply:
Shouldn't the second part go to the description of `send`?